tapfiliate.php

```diff
@@ -154,7 +155,7 @@ function tapfiliate_output_inline_code($is_converting, $external_id_arg = null,
 
     ob_start();
     include(dirname(__FILE__) . '/tracking-snippet.php');
-    $script = ob_get_contents();
+    $script = apply_filters('tapfiliate_snippet', ob_get_contents());
```

Comment:
This should happen in the controller code, in Tapfiliate.php `tapfiliate_output_inline_code`. The options_arg can be the array instead of the JSON encoded version. Then in `tapfiliate_output_inline_code` it can be filtered and JSON encoded.